src/utils/log.ts

```diff
@@ -0,0 +1,14 @@
+import SimpleLogger, { STANDARD_LEVELS } from "simple-node-logger";
+
+const manager = new SimpleLogger();
+
+manager.createConsoleAppender({
+    writer(s: string) {
+        process.stderr.write(s + "\n");
+    },
+});
+
+const logLevelFromEnv = process.env["LOG_LEVEL"] as STANDARD_LEVELS;
+const logger = manager.createLogger(undefined, logLevelFromEnv || "info");
+
+export default logger;
```

Comment:
@nshandra noticed in d2-tools that simple-node-logger works very erratically, we changed there the log module to a custom one. I'll push the changes.